Detector/DetStudies/src/components/DECalAnalysis.h

```diff
@@ -0,0 +1,140 @@
+#ifndef DETSTUDIES_DECalAnalysis_H
+#define DETSTUDIES_DECalAnalysis_H
+
+// GAUDI
+#include "GaudiAlg/GaudiAlgorithm.h"
+#include "GaudiKernel/ServiceHandle.h"
+
+// FCCSW
+#include "FWCore/DataHandle.h"
+#include "TH2F.h"
+#include "TTree.h"
+#include "TF1.h"
+#include "TProfile.h"
+
+class IGeoSvc;
+
+// datamodel
+namespace fcc {
+class PositionedCaloHitCollection;
+class CaloHitCollection;
+class MCParticleCollection;
+}
+
+class TH1F;
+class ITHistSvc;
+/** @class DECalAnalysis DECalAnalysis.h
+ *
+ *  Histograms of energy deposited in active material and total energy deposited in the calorimeter.
+ *  Passive material needs to be marked as sensitive. It needs to be divided into layers (cells) as active material.
+ *  Layers (cells) are numbered starting at 1 so that energy depoisted in cryostat and bath could be easily recognised.
+ *  Sampling fraction is calculated for each layer as the ratio of energy deposited in active material to energy
+ *  deposited in the layer (also in passive material).
+ *
+ *  @author Anna Zaborowska
+ */
+
+class DECalAnalysis : public GaudiAlgorithm {
+public:
+  explicit DECalAnalysis(const std::string&, ISvcLocator*);
+  virtual ~DECalAnalysis();
+  /**  Initialize.
+   *   @return status code
+   */
+  virtual StatusCode initialize() final;
+  /**  Fills the histograms.
+   *   @return status code
+   */
+  virtual StatusCode execute() final;
+  /**  Finalize.
+   *   @return status code
+   */
+  virtual StatusCode finalize() final;
+
+  Double_t FitLongProfile( Double_t *x, Double_t *par);
```

Comment:
Start with lower case. Also, I Suspect it is used within `DECalAnalysis` so it can go to `private`.